### PR TITLE
feat: prompt user wallet when paying an invoice

### PR DESCRIPTION
**Pull request by IgboPharaoh**
The pr introduces a feature for #333 to enable users to select their wallet of choice when paying an invoice. 
- After a user has created an invoice there is a **pay with wallet** button which works in two ways;
   1 -  For mobile devices the user is routed to the play store or app store to download the bitcoin beach wallet or open it from there if already installed. Deep linking to the bitcoin beach wallet and immediately opening it was not possible because you cannot directly access the path to an already installed application because PWAs are installed as web applications rather than native applications which have the ability to use [intents](https://developer.android.com/guide/components/intents-filters#kotlin). 

   2 - For desktop devices, a modal pops up which shows the user the possible wallet options you can use to pay the invoice. You can easily use the bitcoin beach wallet to do this.

image

### CODE REVIEW

**Comment by nicolasburtey** [comment]
didn't skimmed through the PR, but looking at the visual you provide, this looks good. maybe you can add the logo of BBW?

**Comment by IgboPharaoh** [comment]
Alright, that will be done

File: components/ParsePOSPayment/Receive-Invoice.tsx [hunk 1]
```diff
@@ -248,6 +298,59 @@ function ReceiveInvoice({ recipientWalletCurrency, walletId, state, dispatch }:
...
       />
+      {openModal && (
+        <Modal
+          show={openModal}
+          onHide={() => setOpenModal(false)}
+          aria-labelledby="contained-modal-title-vcenter"
+          centered
+        >
+          <Modal.Header className={styles.modal_header} closeButton>
+            You can payment with
```
Comment (Extheoisah) [inline components/ParsePOSPayment/Receive-Invoice.tsx:309]: This should be
```suggestion
            You can make payment with
```
or 
```
You can pay with
```

**Review by Extheoisah — CHANGES_REQUESTED** [review]
This looks good! Just some comments.
Please change the PR title to follow conventional commits https://www.conventionalcommits.org/en/v1.0.0/

File: components/ParsePOSPayment/Receive-Invoice.tsx [hunk 2]
```diff
@@ -248,6 +298,59 @@ function ReceiveInvoice({ recipientWalletCurrency, walletId, state, dispatch }:
...
+          </Modal.Header>
+          <Modal.Body>
+            <div className={styles.modal_item_wrapper}>
+              <Image
+                src="/manifest-logo/logo144x144.svg"
```
Comment (Extheoisah) [inline components/ParsePOSPayment/Receive-Invoice.tsx:314]: Please use the bbw app logo here. this is the cash register's logo
```suggestion
                src="/manifest-logo/logo144x144.svg"
```

**Review by Extheoisah — CHANGES_REQUESTED** [review]

**Comment by IgboPharaoh** [comment]
> didn't skimmed through the PR, but looking at the visual you provide, this looks good. maybe you can add the logo of BBW?



> This looks good! Just some comments. Please change the PR title to follow conventional commits https://www.conventionalcommits.org/en/v1.0.0/

image

**Review by Extheoisah — APPROVED** [review]
looks good. nice work

**Pull request merged**